packages/pds/src/api/com/atproto/repo/importRepo.ts

```diff
@@ -43,10 +44,14 @@ const importRepo = async (
   if (roots.length !== 1) {
     throw new InvalidRequestError('expected one root')
   }
+  const blockMap = new BlockMap()
```

Comment:
We still just read everything into memory for now, but at least it isn't getting hung up on synchronous work.